src/packages/@ncigdc/modern_components/CasesByPrimarySite/CasesByPrimarySite.js

```diff
@@ -0,0 +1,37 @@
+/* @flow */
```

Comment:
I think this file should be renamed. it's more like a disease list right?

Reply:
I'm wondering if I need it at all? I'm just using the relay query in the table model, but not the component.
And yeah, I can rename it, maybe DiseaseListByPrimarySite ?

Reply:
if you're not using it, then yea just remove it